File: Projekat/src/app/entities/friend-request/friend-request.ts
export class FriendRequest {

    ID: number;
    SenderID: string;
    ReceiverID: string;
    Accepted: boolean;
    SenderFirstName: string;
    SenderLastName: string;
    SenderCity: string;

    constructor(id: number, sender: string, receiver: string, accepted: boolean, firstName: string, lastName: string, city: string){
        this.ID = id;
        this.SenderID = sender;
        this.ReceiverID = receiver;
        this.Accepted = accepted;
        this.SenderCity = city;
        this.SenderFirstName = firstName;
        this.SenderLastName = lastName;
    }
}

File: ProjekatApi/ProjekatApi/Model/FriendRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace ProjekatApi.Model
{
    public class FriendRequest
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string SenderID { get; set; }
        public string ReceiverID { get; set; }
        public bool Accepted { get; set; }
        public string SenderFirstName { get; set; }
        public string SenderLastName { get; set; }
        public string SenderCity { get; set; }
    }
}
